fearless/Common/View/UIFactory.swift

```diff
@@ -187,6 +190,14 @@ final class UIFactory: UIFactoryProtocol {
         return view
     }
 
+    func createBorderedContainerView() -> BorderedContainerView {
+        let view = BorderedContainerView()
+        view.borderType = .bottom
+        view.strokeWidth = 1.0 / UIScreen.main.scale
```

Comment:
is defined in UIConstants.separatorHeight

Reply:
fixed